examples/Getting_Started/Getting-started-with-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,687 @@
+{
```

Comment:
<p>"After we load the workflow..." (Googling indicates that "have loaded" is the present perfect tense.  My $0.02 is that present tense is perfectly clear.)</p><p> </p><p>I appreciate that you indicated why the action is necessary and helpful.  Thank you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>